CalibTracker/SiPixelConnectivity/test/read_cfg.py

```diff
@@ -0,0 +1,66 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("SiPixelCablingReader")
+process.load("Geometry.CMSCommonData.cmsIdealGeometryXML_cfi")
+process.load("Geometry.TrackerGeometryBuilder.trackerGeometry_cfi")
+process.load("Geometry.TrackerNumberingBuilder.trackerNumberingGeometry_cfi")
+process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
+process.GlobalTag.globaltag = 'PRE_MC_71_V5::All'
```

Comment:
@dkotlins, there is no possibility anymore to read from V1 in 80X, please change the GT to be compliant with the new DB version